templates/base.html

```diff
@@ -8,35 +8,32 @@
     <!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
 
     <link rel="stylesheet" href="{% static "css/bootstrap.min.css" %}" type="text/css" media="screen" />
-    <link rel="shortcut icon" href="{% static "img/favicon.ico" %}" />
-    <link href="https://fonts.googleapis.com/css?family=Libre+Franklin" rel="stylesheet">
+    <link rel="icon" href="{% static 'images/favicon-32x32.png' %}" sizes="32x32"/>
+    <link rel="icon" href="{% static 'images/favicon-16x16.png' %}" sizes="16x16"/>
+    <link href="https://fonts.googleapis.com/css?family=Libre+Franklin|Lato:900" rel="stylesheet">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.2/css/all.css" />
-    <link rel="stylesheet" href="{% static "css/custom.css" %}" type="text/css" />
     {% block extra_css %}{% endblock %}
+    <link rel="stylesheet" href="{% static "css/custom.css" %}" type="text/css" />
 </head>
 <body>
 
     <nav class="navbar navbar-expand-lg">
-        <a class='navbar-brand' href="{% url 'index' %}">
-            <img src="{% static 'images/connect-icon.png' %}" class="img-fluid" width="110px">
-            <!-- Connect to End Homelessness -->
-        </a>
         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
There is some funky coloring on the responsive view of the navbar. I think the hamburger icon is white, which is why it isn't appearing. (Highlighting is my own).
<img width="355" alt="screen shot 2019-01-09 at 10 28 54 am" src="https://user-images.githubusercontent.com/6961258/50913450-03eb1300-13fa-11e9-9d86-367b410f5b81.png">
<img width="330" alt="screen shot 2019-01-09 at 10 29 06 am" src="https://user-images.githubusercontent.com/6961258/50913451-03eb1300-13fa-11e9-81cb-c66308e1442b.png">